client/src/Components/Questions.js

```diff
@@ -120,96 +134,91 @@ const Questions = (props) => {
 							id="answer_a"
 							checked={isChecked && radioId == "answer_a" ? true : false}
 						/>
-					) : null}
-					{answer_a ? <Label for="answer_a">{answer_a}</Label> : null}
-				</FormGroup>
+						 <Label for="answer_a">{answer_a}</Label></FormGroup>) : null}
 
-				<FormGroup className="answers">
-					{answer_b ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_b}
-							onChange={checkHandler}
-							id="answer_b"
-							checked={isChecked && radioId == "answer_b" ? true : false}
-						/>
-					) : null}
-					{answer_b ? <Label for="answer_b">{answer_b}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_c ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_c}
-							onChange={checkHandler}
-							id="answer_c"
-							checked={isChecked && radioId == "answer_c" ? true : false}
-						/>
-					) : null}
-					{answer_c ? <Label for="answer_c">{answer_c}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_d ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_d}
-							onChange={checkHandler}
-							id="answer_d"
-							checked={isChecked && radioId == "answer_d" ? true : false}
-						/>
-					) : null}
-					{answer_d ? <Label for="answer_d">{answer_d}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_e ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_e}
-							onChange={checkHandler}
-							id="answer_e"
-							checked={isChecked && radioId == "answer_e" ? true : false}
-						/>
-					) : null}
-					{answer_e ? <Label for="answer_e">{answer_e}</Label> : null}
-				</FormGroup>
-				<FormGroup className="answers">
-					{answer_f ? (
-						<Input
-							type="radio"
-							name="answer"
-							value={answer_f}
-							onClick={checkHandler}
-							id="answer_f"
-							checked={isChecked && radioId == "answer_f" ? true : false}
-						/>
-					) : null}
-					{answer_f ? <Label for="answer_f">{answer_f}</Label> : null}
+
+					<FormGroup className="answers">
+						{answer_b ? (
+							<Input
+								type="radio"
+								name="answer"
+								value={answer_b}
+								onChange={checkHandler}
+								id="answer_b"
+								checked={isChecked && radioId == "answer_b" ? true : false}
+							/>
```

Comment:
you should use === 
it's not complain but == is not correct

Reply:
agree

Reply:
by the way i just notced that's not my code